packages/wallet-protocols/src/protocols/create-null-channel/protocol.ts

```diff
@@ -1,10 +1,12 @@
 import { assign, Machine } from 'xstate';
 import { HashZero, AddressZero } from 'ethers/constants';
+import { Outcome } from '@statechannels/nitro-protocol';
```

Comment:
This is an unused import: do we want  to add the `"noUnusedLocals": true,` option to `tsconfig.ts` for this package? Most of the other packages have it.

Reply:
We definitely want to add it!